### DISCUSSION TITLE
大一CPR實體技術考試改制真的賊舒服

### DISCUSSION THREAD

**Discussion by lyykey** [1101-GS4719-C/Python-Community-GS4719-C#1915] [General]
只要你是中央大學生，大學畢業前你都要參加大一CPR實體技術考試。
在過去的五學期，我都落選大一CPR實體技術考試，讓我備受挫折，深怕自己畢不了業。怎麼選都選不上，都不知道是學校問題還是自己網路真的太爛，一直在問候學校伺服器爸媽。
不過！我大中央終於變通了，新體制的大一CPR實體技術考試讓我們人人皆有機會報名，而且就算到現在甚至都還有空位，如下圖。
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/105580604/2dfe7ae3-81e3-43a3-9468-6bf59125a81d)
歡迎還沒報名的同學趕緊手刀報名。
祝大家學業順利，不必再為大學瑣事干預大學生活~

**Comment by benbebobo149** [comment]
我覺得這個措施非常的棒，因為學校根本沒有足夠的師資可以訓練所有的學生，每一次都要花很多時間膽戰心驚的再電腦面前搶票。但我覺得可以從另一個方面去想，如果線上上課，真的會讓學生了解做出正確的CPR嗎?因為這真的是一個很重要的技能，哪一天你身邊的家人朋友出事了，做了CPR或許就可以拯救他們的性命，但是這樣的政策，會讓學生對於CPR課程淪為一個考試科目，掛網修課的事情頻傳。我認為學校應該要另想其他辦法，讓所有學生可以實體並且沒有壓力的修到這門課程。最後，我要感謝同學分享這個讓我思考的議題。

**Comment by 155Julia** [comment]
這樣的機會真的太棒了！感謝同學的分享，讓我們知道還有額滿的場次，我剛剛也手刀報名了。這種實際技能的學習對我們來說非常重要，不僅可以提高對於生命安全的警覺性，還可以在緊急情況下保持冷靜、有效地應對。期待和同學們一起在考試中學習和成長！

**Comment by sunset513** [comment]
當我閱讀這篇文章時，突然湧上心頭的是當年我們宿舍一起搶救心跳呼吸停止的緊張情景。回想起當時的場景，彷彿又身臨其境，感受到了那份緊迫而又壓抑的氛圍。我記得當時大家都全力以赴，緊張地合作著進行心肺復甦，每一個動作都充滿了決心與希望。那一刻，我們彼此間的連結更加緊密，共同面對挑戰的過程中，我們彼此的情感和信任得到了更深層次的凝聚。這段回憶激發了我對友誼和團隊合作的思考，讓我更加珍惜當時的經歷和我們之間的深厚情誼。

**Comment by wanderer657** [comment]
雖然個人已經參與過CPR了，但當年也是在系館跟著一群系上同學一起搶著報名過，每當想起此事，或者看到至今仍在參與的室友，就會讓我感慨好險中央有對此做出改變，使心肺復甦的技巧更普及，除了應對緊急狀況外，也讓校內每個參與過CPR的同學產生出相互關懷，注意他人的良善感，我認為讓更多人能參與到實體技術考試是十分關鍵的。

**Comment by yyujimmy** [comment]
我覺得學習CPR對大學生來說是一個非常實用且有價值的學習經驗。首先，CPR是一項關鍵的急救技能，對於處理意外事件或突發緊急情況至關重要。透過大學課程學習CPR，學生能夠獲得正確的知識和技能，提高應對緊急情況的能力。

此外，CPR課程也有助於擴展學生的健康和醫療知識。通過學習CPR，學生不僅能夠學會如何應對心臟病發作、窒息等急救情況，還能夠理解人體生命支持系統的基本原理，提高對身體健康和安全的認識。

最重要的是，學習CPR有助於培養學生的團隊合作和社會責任感。在課程中，學生需要與他人合作進行模擬演練，這不僅促進了他們的團隊合作能力，還讓他們意識到自己有能力在緊急情況下為他人提供幫助，培養了樂於助人的精神和社會責任感。

總體而言，學習CPR對大學生來說是一個極具價值的學習體驗，不僅可以提高應對緊急情況的能力，還能夠擴展健康和醫療知識，並培養團隊合作和社會責任感。這樣的學習經驗對於他們的個人成長和社會貢獻都有著積極的影響。

**Comment by YANRONG0123** [comment]
看到這篇文章，想起我以前和室友在大一上學期的時候，早上5.多爬起來搶CPR，最後當然是有成功，且四人一起參與同一個時段，在之後打聽之下才發現，原來能準時實現「大一」CPR，真的是三生有幸，在往後幾年也可以不用為了這個畢業們門檻而煩惱。

**Comment by hbh10601** [comment]
確實，我上學期的時候去，超過預計時間1個多小時，而且一開始要先上先備知識課，然後上完課後，所有人再一個一個排隊考試，沒考過的還要再重考。現在這樣分流確實方便很多，我那時候還餓到肚子痛qq

**Comment by wuyu06** [comment]
在看到這篇文章前，我還以為我要等到大四才能搶到ＣＰＲ，不過除了學校的考試外，參加急救社也能抵免

**Comment by woohsin** [comment]
白癡事件分享，這次搶到去看才發現其實大1有搶到但沒有發現，白白浪費...

**Comment by 110101501** [comment]
自己蠻幸運的大一上就搶到名額，就提前做完cpr了。
在考試還沒改制之前每次的cpr 都會搶破頭，聽身邊的朋友同學分享的經驗不外乎就是沒有搶到。
改制是好事，能分流、讓大家更能儘早完成畢業條件。

**Comment by thomasxshen** [comment]
真的覺得大一CPR超級垃圾，規定必修又沒有建全的配套措施，搞得選個大一CPR都像在抽獎一樣，我是直到大二下才抽到，還是很早起去搶的，能把一門課搞到要早起去搶真的點點點，更何況不修還不能畢業，一堆人直到大四才修到真的覺得很可悲，一方面覺得我們學校終於懂得變通了有點欣慰，另一方面又覺得這又不是件很難解決的事情還拖這麼久才處理而對我們學校的行政效率感到可悲。

**Comment by willychiuchiu** [comment]
每次只要到搶cpr的日子，就好比搶演唱會票一樣，有夠難搶！伺服器都一定要掛個20分鐘才能進去網站，真的讓人很煎熬，名叫大一cpr但根本沒幾個人能在大一就搶到，這制度真的很有問題。現在改制才終於變得比較合理，不過我好不容易上學期已經通過所以也不乾我的事了嗚嗚

**Comment by yxin214** [comment]
第一次搶CPR時，和同學們到處尋找學校網路最好的地方，並在時間一到拼命刷新頁面，最後還是沒能搶到名額。改制後狀況明顯好很多，用影片教學不但能在不熟的地方重複觀看，也能讓大家不受時地影響的分配學習進度。

**Comment by Mitochondrionjessica** [comment]
真的超讚! 改制除了每場考試可以增加更多學生名額外，上課效率也變高。可以利用通勤瑣碎的時間完成CPR的線上課程，不用上完一整天課還要實體上課到晚上十點。

**Comment by William08290829** [comment]
我覺得大一的CPR課程挺有用的。上過一次課程你就知道怎麼在緊急情況下給人做心肺復蘇。這不僅讓你多了一項技能，而且也讓你有種保護自己和別人的能力。但因為是大一的時候上的，老實說到現在有些細節現在都快忘光了。

**Comment by Eileen050824** [comment]
我認真覺得這個改變很棒，主要是大家國高中勢必經歷過類似課程訓練，能夠把一些補強資源用網路分攤，我認為是相當明智且大幅提升效率的選擇。此外，學校的CPR報名網頁實在是不予置評(有搶過的都懂)，縮短考試時間並增加應試時段，真的有讓學生真的更方便了！

**Comment by wanka2030** [comment]
只能說上學期還在那邊搶資格的我成為了一個小丑🤡上學期剛下課就馬上打開手機搶，學校的網頁還癱瘓，但五分鐘之後名額也都沒了（根本在搶小巨蛋，這學期反而去挑一些死人時間，做個3分鐘就輕輕鬆鬆完成畢業條件了✅

**Comment by jim1743** [comment]
我是很幸運的早早大一就搶到CPR的課程，我也認為這次的改制真的很不錯，能讓其他還沒上過的同學有更便捷的方式選到CPR課程。我認為CPR在發生緊急狀況的時候很有幫助，讓自己多一項救助別人的技能。

**Comment by poiu6fi** [comment]
我也覺得原本那種報名的方式真的很爛，每次都搶到破頭結果還是搶不到
